docs/performance/selects.txt

```diff
@@ -0,0 +1,88 @@
+==================
+Select Performance
+==================
+
+Aggregations and Group By
+=========================
+
+It is common to do ``GROUP BY`` queries on tables for analytics purposes. For
+example, you might select the ``avg``, ``max``, and ``min`` of some
+measurements over a billion records and group them by device ID.
+
+By default, CrateDB processes all matching records. This may require a lot of
+processing power, depending on the data set and the size of the CrateDB
+cluster.
+
+Similar products sometimes limit the amount of records that are processed for
+``GROUP BY`` operations. That means the user might get inaccurate results, but
+for analytics this is often an acceptable trade-off.
+
+To emulate this down sampling behavior, users can filter on the ``_docid``
```

Comment:
I think we're jumping in too early here and we might need to unpack this a bit so readers have more context/understanding

same is true for the rest of the doc so will hold off leaving further comments until the previous para is expanded and I have a better understanding myself